frontend/src/concepts/pipelines/elyra/utils.ts

```diff
@@ -86,7 +87,10 @@ export const generateElyraSecret = (
         auth_type: 'KUBERNETES_SERVICE_ACCOUNT_TOKEN',
         api_endpoint: route,
         // Append the id on the end to navigate to the details page for that PipelineRun
-        [ELYRA_SECRET_DATA_ENDPOINT]: `${location.origin}/pipelineRuns/${namespace}/pipelineRun/view/`,
+        [ELYRA_SECRET_DATA_ENDPOINT]: `${location.origin}${routePipelineRunDetailsNamespace(
```

Comment:
is there a test for this change?

Reply:
Hmm, no... Any suggestion on how I can write a test on this?

Reply:
You could unit test `generateElyraSecret`